.github/workflows/actionlint.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           # SC2086 - Double quote to prevent globbing and word splitting. - https://www.shellcheck.net/wiki/SC2086
           SHELLCHECK_OPTS: --exclude=SC2046,SC2086
         with:
-          fail_on_error: true
+          # fail_level: error
```

Comment:
`fail_on_error: true` is deprecated, the correct way is `fail_level: error`. Commented out for now to see whether the thing actionlint complains about would even work.

Reply:
We need to not comment this out before merging.